mlir/include/mlir/IR/Constraints.td

```diff
@@ -153,6 +153,9 @@ class TypeConstraint<Pred predicate, string summary = "",
     Constraint<predicate, summary> {
   // The name of the C++ Type class if known, or Type if not.
   string cppClassName = cppClassNameParam;
+  // TODO: This field is sometimes called `cppClassName` and sometimes
```

Comment:
This one has cppClassName above, is this effectively duplicating it to enable reuse? (Alternative would be having ODS check for either as staging mechanism + no need to introduce here, then full rename can be done)

Reply:
(having seen usage, what you have done look like less changes)

Reply:
Fixed in #102657.